src/blockbroadcast.jl

```diff
@@ -79,7 +79,6 @@ julia> cumulsize = [1, 2, 4, 5, 7];
 julia> for idx in SubBlockIterator(subcumulsize, cumulsize)
            B = @show view(A, idx)
            @assert !(parent(B) isa BlockArray)
-
            idx :: BlockIndexRange
```

Comment:
As I reported in https://github.com/JuliaDocs/Documenter.jl/issues/963, it looks like `for` loop can't contain empty lines ATM.